layouts/shortcodes/answer-key/q.html

```diff
@@ -0,0 +1,32 @@
+{{ $q := .Get "question" }}
+{{ $id := .Get "id" }}
+
+
+<div
+  x-data="{
+        id: {{ $id }},
+        get expanded() {
+            return this.active === this.id
+        },
+        set expanded(value) {
+            this.active = value ? this.id : null
+        },
+    }"
+  role="region"
+>
```

Comment:
This will soon be doable in native HTML with no JS using `<details name="groupname">`, but it's not in Firefox yet, alas. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details#name